File: test/interface/golden/__tests__/product/OneTypeParameter_spec.ml
let () =
  AesonSpec.sampleGoldenAndServerSpec (OneTypeParameter.decodeOneTypeParameter (Aeson.Decode.wrapResult Aeson.Decode.int)) (OneTypeParameter.encodeOneTypeParameter Aeson.Encode.int) "oneTypeParameter" "http://localhost:8081/OneTypeParameter/OneTypeParameter" "golden/product/OneTypeParameter.json";

File: test/interface/golden/lib/js/__tests__/product/OneTypeParameter_spec.js
'use strict';

var AesonSpec        = require("bs-aeson-spec/lib/js/src/AesonSpec.js");
var OneTypeParameter = require("../../product/OneTypeParameter.js");

AesonSpec.sampleGoldenAndServerSpec((function (param) {
        return OneTypeParameter.decodeOneTypeParameter(AesonSpec.decodeIntWithResult, param);
      }), (function (param) {
        return OneTypeParameter.encodeOneTypeParameter((function (prim) {
                      return prim;
                    }), param);
      }), "oneTypeParameter", "http://localhost:8081/OneTypeParameter/OneTypeParameter", "golden/product/OneTypeParameter");

/*  Not a pure module */
